packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -79,8 +78,7 @@ export function withResponsiveMode<TProps extends { responsiveMode?: ResponsiveM
 
     private _getResponsiveMode(): ResponsiveMode {
       let responsiveMode = ResponsiveMode.small;
-      const element = findDOMNode(this) as Element;
-      const win = getWindow(element);
+      const win = getWindow();
```

Comment:
isn't this a breaking change for multi-window scenario?

Reply:
As I called out in my email, yes. I am incorporating changes from #11870 to account for this. Namely, the export of a setter to invoke prior to mount of `withResponsiveMode`.

Do you have alternative recommendations I should consider?

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/pull/12258/commits/e6454e02c8a936bbdace936fb4065d2654f52a68 requires the caller to avoid the unnecessary render while still supporting the multi-window scenario.